safe-global/safe-wallet-web
# Remove module flow

## What it solves

Part of #2067 

## How this PR fixes it

- Moves Remove Module to the new TxFlow

## How to test it

1. Open a Safe with a module
2. Remove the module
3. Observe the new flow

## Screenshots
<img width="1126" alt="Screenshot 2023-06-13 at 16 34 21" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/db2b0356-7f95-428f-8106-e4ff2d801cbc">

## Checklist
